### PR TITLE
add xstar

### PR DESCRIPTION
**Pull request by zerokarafont**
**NOTE**

#### Please enable "Allow edits by maintainers" while putting up the PR.

---

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
4. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
5. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
6. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Short Description (to be shown on DefiLlama):
XSTAR, the Omnichain Identity Protocol for Proof of Humanity, features an adaptive humanity scoring system that securely and privately authenticates individual.

##### Implementation Details: Briefly describe how the oracle is integrated into your project:
The Chainlink oracle data account address is configured in our Oracle contract. When a user sends a transaction, the smart contract retrieves price data from the configured price data account and performs the necessary calculations.

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/xstar exports TVL: 
        
 
 
    btr                       0
    
    total                    0

File: projects/xstar/index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const { sumTokens2, } = require('../helper/unwrapLPs')
+
+async function tvl(api) {
+  
+  return sumTokens2({ owners: [
+    '0xFEc8a6cFaCbF3dA3E0c9988D59542eB6A5d49184',
```
Comment (g1nt0ki) [inline projects/xstar/index.js:6]: hi @zerokarafont what is this contract? can users withdraw funds from it?

Comment (zerokarafont) [reply projects/xstar/index.js:6]: this is controller contract, accept native currency for income, only deployer can withdraw funds

Comment (g1nt0ki) [reply projects/xstar/index.js:6]: hmm, we cant count it as tvl since users cant withdraw funds

**Pull request closed without merging**